packages/react/src/Heading/index.ts

```diff
@@ -1,2 +1,3 @@
-export { Heading } from './Heading'
-export type { HeadingProps } from './Heading'
+export type { HeadingLevel, HeadingProps } from './Heading'
+
+export { Heading, getHeadingElement } from './Heading'
```

Comment:
Same thing for `getHeadingElement`, when would a consumer need this?

Reply:
Not exporting made the unit tests for Accordion fail – although I must say I didn’t investigate very deeply _why_ these tests seem to need access to this function.

Reply:
Updated according to our discussion leading to #409. Test still working.